src/obs_inv_utils/aws_s3_interface.py

```diff
@@ -0,0 +1,243 @@
+import re
```

Comment:
suggestion on the naming of this file "src/obs_inv_utils/aws_s3_interface.py"
should we name all interfaces like "interface_aws_s3", "interface_hpss"

Reply:
After looking at this again, I remembered why I chose that naming convention.  It follows the conventions set by UFS-RNR.  Namely, [specificity]_[method_type].py  I'd like to keep this convention if it's ok with the group.